Find a functionally equivalent code:android.media.MediaPlayer.TimeProvider.onSeekComplete:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,5 +1,7 @@
 {
     synchronized (this) {
+        mStopped = false;
+        mSeeking = true;
         scheduleNotification(NOTIFY_SEEK, 0);
     }
 }

